aspnetcore/fundamentals/portable-object-localization.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Configure Portable Object localization
+author: sebastienros
+description: This article introduces Portable Object files and outlines the necessary steps for using them in an ASP.NET Core application.
+keywords: ASP.NET Core,localization,culture,language,portable object
+ms.author: scaddie
+manager: wpickett
+ms.date: 09/18/2017
+ms.topic: article
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: fundamentals/portable-object-localization
+---
+# Configure Portable Object localization in ASP.NET Core
+
+By [Sébastien Ros](https://github.com/sebastienros) and [Scott Addie](https://twitter.com/Scott_Addie)
+
+This article walks through the steps for using Portable Object (PO) files in an ASP.NET Core application.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/localization/sample/POLocalization)
+
+## What is a PO file?
+
+PO files contain the translated strings for a given language. They prove very useful, as opposed to standard *.resx* files. PO files support pluralization and are distributed as plain text files.
```

Comment:
>They prove very useful, as opposed to standard *.resx* files.

Why? Why are you saying .resx files are not useful? ~very~ is dead wood but I think this sentence needs to go. We need a comparison between the two approaches.

Reply:
This is not what I wrote initially, "They reveal very useful as contrary to standard resx files, PO files support pluralization and are distributed as text files." But I understand how the information got lost because of my grammar. Here you should read "They are very useful" **AS** "contrary to standard resx files, PO files support pluralization and are distributed as text files" and this is very important in the context of this article.